### ISSUE TITLE
Unable to build jruby-head: old maven version

### ISSUE THREAD

**Issue by donv** [rvm/rvm#3499]
Hi!

Currently jruby-head fails to build using RVM stable or head.  The included version of maven is 3.1.1, but version 3.3.3 or newer is needed.

```
$ which mvn
~/.rvm/bin/mvn
$ mvn --version
Apache Maven 3.1.1 (0728685237757ffbf44136acec0402957f723d9a; 2013-09-17 17:22:22+0200)
Maven home: /home/capistrano/.rvm/java/apache-maven-3.1.1
Java version: 1.8.0_45, vendor: Oracle Corporation
Java home: /usr/lib/jvm/java-1.8.0-openjdk-1.8.0.45-30.b13.el7_1.x86_64/jre
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "3.10.0-123.20.1.el7.x86_64", arch: "amd64", family: "unix"
$ rvm --version
rvm 1.26.11 (master) by Wayne E. Seguin <wayneeseguin@gmail.com>, Michal Papis <mpapis@gmail.com> [https://rvm.io/]
```

**Comment by donv** [comment]
Hi!

I did a

    rvm get branch mvnw

and

    rvm reinstall jruby-head

and it completed OK and the installed jruby-head looks good.

Thank you!

**Comment by havenwood** [comment]
Closed by #3502.
